projects/storefrontstyles/scss/components/layout/header/_site-context-selector.scss

```diff
@@ -36,6 +36,9 @@ $caretSize: 7px;
     @include forFeature('a11yShowDownArrowOnFocusedSelectMenu') {
       z-index: 1000;
     }
+    @include forFeature('a11yVisibleFocusOverflows') {
```

Comment:
There is a z-index here for the feature flag just before this one. Should we change that index to `1` too?